signbank/dictionary/models.py

```diff
@@ -810,25 +810,17 @@ class SignSeenChoices(models.TextChoices):
         help_text="Result of the survey question 'Have seen it or use it myself'"
     )
     response_id = models.CharField(
-        max_length=255, help_text="Identifier of specific survey result in Qualitrics"
+        max_length=255, help_text="Identifier of specific survey result in Qualtrics"
     )  # can potentially make this unique
     respondent_first_name = models.CharField(
         max_length=255, default="", help_text="Survey respondents first name"
     )
     respondent_last_name = models.CharField(
         max_length=255, default="", help_text="Survey respondents last name"
     )
-    respondent_email = models.EmailField(default="", help_text="Survey respondents email")
```

Comment:
**question**: assuming this was cleared by Micky & they'll get in touch by looking directly at the Qualtrics data?

Reply:
yes, exactly that